akka-remote/src/main/scala/akka/remote/artery/Control.scala

```diff
@@ -29,22 +29,22 @@ private[akka] trait Reply extends ControlMessage
  * Marker trait for control messages that can be sent via the system message sub-channel
  * but don't need full reliable delivery. E.g. `HandshakeReq` and `Reply`.
  */
-private[akka] trait ControlMessage
+private[akka] trait ControlMessage extends Serializable
```

Comment:
To be able to override default (Java) serialization in the config without having to list all subclasses of ControlMessage separately.